Sources/FluentUI_common/Core/Theme/Tokens/GlobalTokens.swift

```diff
@@ -1796,6 +1796,7 @@ public class GlobalTokens: NSObject {
         case radius60
         case radius80
         case radius120
+        case radius400
```

Comment:
Would love to get input from @joannaquu and @mischreiber here - naively I would want to avoid having to add something to GlobalTokens (cost/complexity to every user of any controls) which is only used in the BottomSheet control.

But 2 potential considerations which could counter:
1. Do we expect to use this in more places as we adapt more of the design to iOS 26 and this is just the first step?
2. Is there something about the tokens system where forcing these to be added globally is better than just defining values in each TokenSet?

Reply:
Agreed that this is not something that should live in the GlobalTokens. It's not currently part of the Fluent design language, and shouldn't be added here unless we decide to update that. Instead, let's store the constant in the `BottomSheetTokenSet` for now and reevaluate as we implement OS26 support.

Reply:
tbh I never super understood why we had global tokens for ints. maybe just consistency? but I'll defer to Mike

Reply:
oops he's already responded